test/libethereum/Block.cpp

```diff
@@ -50,6 +50,8 @@ BOOST_AUTO_TEST_CASE(bStructures)
 
 BOOST_AUTO_TEST_CASE(bStates)
 {
+	if (!dev::test::Options::get().quadratic)
```

Comment:
I think "labels" is what we want in this case: http://www.boost.org/doc/libs/1_63_0/libs/test/doc/html/boost_test/tests_organization/tests_grouping.html

They are not dynamic (external config file), but at least we don't need to create our version of it.
But that should go into a separated PR.